etc/orthography.tsv

```diff
@@ -0,0 +1,360 @@
+Grapheme	IPA	CLTS	Examples
+*	NULL	?	natriːn*
++	NULL	?	z+C:AUumgeː
+?	NULL	?	awər_?, kyjøŋ?, ɑːli_mambat͡sa?_maːʃiː
+A	?	?	z+C:AUumgeː
+C:	tʃ	voiceless post-alveolar sibilant affricate consonant	z+C:AUumgeː
+Grapheme	IPA	CLTS	Examples
+Is	?	?	Is
+U	?	?	z+C:AUumgeː
+[	NULL	?	[djuː_sat], kijuːɻ_toːt_[kʲyːɻ], maːmas_paːnt͡ʃe_[päːnje]
+]	NULL	?	[djuː_sat], kijuːɻ_toːt_[kʲyːɻ], maːmas_paːnt͡ʃe_[päːnje]
+_	+	?	[djuː_sat], abaw_aʒdaːd, abeːni_braː
+ʰl	ʰl	?	pit͡ʃ_ʰlag_pẽːɽ, pit͡ʃ_ʰlag_praː
+a	a	unrounded open front vowel	[djuː_sat], a, aba
+ae	a e	from unrounded open front to unrounded close-mid front diphthong	awar_laem, awar_saem, aːiː_tarapeː_d͡ʒudulaem
+aeː	a eː	unrounded open front vowel	awar_laeː, aʈʰaeːʃ, aʈʰaeːʃu_biːʃ
+aẽ	a ẽ	unrounded open front vowel	paẽ
+aẽː	a ẽː	unrounded open front vowel	hãɽaẽː
+ai	ai	from unrounded open front to unrounded close front diphthong	ai, aid, akai_kata
+aiː	a iː	unrounded open front vowel	aiːk, daiː, daiː_ga_biː
+aĩ	a ĩ	from unrounded open front to nasalized unrounded close front diphthong	paĩʃi, paĩʃo_daʂ, taĩ_uːlaːd
+aĩː	a ĩː	unrounded open front vowel	habaĩː, habaĩː_mʰaːl, kẽːʈʰaĩː
+aoː	a oː	unrounded open front vowel	jaoːlas
+au	au	from unrounded open front to rounded close back diphthong	audio:_bɻoː_ɻu, bau, jau
+auː	a uː	from unrounded open front to long rounded close back diphthong	d͡ʒauːɬ, jauːlas
+aː	aː	long unrounded open front vowel	abaw_aʒdaːd, abawaʒdaːt, abaː
+aːe	aː e	unrounded open front vowel	akaːe, amaːesam_we, baːe
+aːeː	aː eː	unrounded open front vowel	gəɳaniː_sajaːeː, t͡ʃənɖiː_sajaːeː
+aːẽ	aː ẽː	unrounded open front vowel	ãʂʈaːẽ, biho_ãʂʈaːẽ, biho_sataːẽ
+aːi	aːi	from long unrounded open front to unrounded close front diphthong	awar_laːi, awlaː_laːi, awər_aːi
+aːiː	aː iː	unrounded open front vowel	awar_aːiː, aʈʰaːiː, aːiː
+aːu	aːu	from long unrounded open front to rounded close back diphthong	aːu, aːu_ami, blaːuɬ
```

Comment:
It is not clear if the languages are better diagnosed as having dipthongs or two vowels in these instances here. Note that this is information that the expert needs to provide. My criterion would be if both vowels bear the same "weight". German "Eier" would be "ai ə", as it counts as two syllable, so "ai" is a dipthong.